src/components/elements/text/types.ts

```diff
@@ -16,6 +16,6 @@ export type TTextType =
 	| "text-bodysmall"
 	| "text-xsmall";
 
-export interface ITextSchema extends IBaseElementSchema<TTextType>, TComponentOmitProps<V2_TextProps, "children"> {
+export interface ITextSchema extends IBaseElementSchema<TTextType>, TComponentOmitProps<TypographyProps, "children"> {
```

Comment:
should still expose the old prop values for weight to ensure backwards compatibility in the types

Reply:
this element should be marked as deprecated as well, as consumers should switch to the new `typography` element